WcaOnRails/spec/features/sign_up_spec.rb

```diff
@@ -82,6 +83,75 @@
 
       expect(page).to have_selector('#have-competed', visible: true)
     end
+
+    it "remembers their selected wca id on validation error" do
+      visit "/users/sign_up"
+      click_on "I have competed in a WCA competition."
+      # They have not selected a valid WCA ID yet, so don't show the birthdate verification
+      # field.
+      expect(page.find("div.user_dob_verification", visible: false).visible?).to eq false
```

Comment:
Basing on a line below, couldn't you use this everywhere?
```ruby
expect(page).to have_selector("div.user_dob_verification", visible: false)
```
The current approach redundant =)